gitlab/__init__.py

```diff
@@ -26,6 +26,9 @@
 import requests
 import six
 
+import logging
+logger = logging.getLogger(__name__)
```

Comment:
Could you remove the logging code for this change? I'm not against adding some logging but I'd rather do it in another patch.